### PR TITLE
Import/export docs

### LINKED ISSUE #2783: Document importing/exporting data with Airlock for Researchers
Adding a section on "Importing and exporting data" in https://microsoft.github.io/AzureTRE/using-tre/tre-for-research/ which would be a user-facing doc describing the process of importing and exporting data from a researcher's point of view

This should include guidance on
- [x] Creating a draft request
- [x] Generating a SAS and examples of different methods to add blob (CLI & Storage Explorer)
- [x] Submitting the request
- [x] Viewing status of a request
- [x] Accessing the output file

### PR DESCRIPTION
**Pull request by jjgriff93**
# Resolves #2783

## What is being addressed

Add documentation for researchers with steps on importing and exporting data using airlock.

### CODE REVIEW

File: docs/using-tre/tre-for-research/importing-exporting-data.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+[![Get storage link](../../assets/get-request-storage-link.png)](../../assets/get-request-storage-link.png)
+
+3. Copy the URL and use it to upload your data to the Azure Storage account. You can use several tools for this that accept SAS URLs, such as the Azure Storage Explorer, or the Azure CLI, depending on your preference.
+
+    - To use Storage Explorer, follow [this guide](https://learn.microsoft.com/en-us/azure/vs-azure-tools-storage-manage-with-storage-explorer?tabs=macos)
+
+    - With the Azure CLI, you can run `az storage blob upload -f /path/to/file --blob-url SAS_URL`. [More info](https://learn.microsoft.com/en-us/cli/azure/storage/blob?view=azure-cli-latest#az-storage-blob-upload)
+
+!!! warning
+    Airlock only supports single files per request. If you need to import multiple files, please zip them before uploading to the request's storage container.
```
Comment (tanya-borisova) [inline docs/using-tre/tre-for-research/importing-exporting-data.md:44]: nit: single file?

File: docs/using-tre/tre-for-research/importing-exporting-data.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+1. Open your TRE UI and navigate to the workspace you wish to export data from
+
+2. Navigate to the Airlock tab (in the left-hand menu) and click *Create new*, then select *Export*
+
+3. Fill in a suitable **Title** and **Business Justification** for the request then hit *Create*
+
+4. Once the draft request pop-out opens, click *Generate* in the **Files** section to generate a Storage SAS URL to use for uploading your data.
+
+5. You now need to head into your Workspace VM/resource containing the data you wish to export, and paste in the SAS URL you've just generated. Use your preferred storage tool to upload the data to the request container. See Step 2 in the [Importing data](#importing-data-to-a-workspace) section for more details on examples of these tools
```
Comment (tanya-borisova) [inline docs/using-tre/tre-for-research/importing-exporting-data.md:84]: nit: maybe "for more details on using these tools" or "for more examples of these tools"?

**Review by tanya-borisova — APPROVED** [review]
Looks great! Clear and concise 🚀

**Comment by marrobi** [comment]
@jjgriff93 this needs adding to the mkdocs.yml file otherwise the files wont show

**Comment by jjgriff93** [comment]
> @jjgriff93 this needs adding to the mkdocs.yml file otherwise the files wont show

Dangnamit - knew I forgot something, thanks - will add a quick new PR

**Pull request merged**